.github/workflows/progress-tracker.yml

```diff
@@ -0,0 +1,126 @@
+name: Progress Tracker
+
+env:
+  # Update these environment variables every year
+  FILTER_LABEL: '2020 chapter'
+  TRACKER_ISSUE_NUMBER: 989
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    name: Track Progress
+    if: github.repository == 'HTTPArchive/almanac.httparchive.org'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.issue.labels.*.name, env.FILTER_LABEL)
+        with:
+          script: |
+            const FILTER_LABEL = process.env.FILTER_LABEL
+            const TRACKER_ISSUE_NUMBER = process.env.TRACKER_ISSUE_NUMBER
+
+            const parseTasks = text => {
+              const tasks = []
+              text.split('\n').forEach(line => {
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[ \]\s+\S+/.test(line)) {
+                  tasks.push('')
+                }
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[[xX]\]\s+\S+/.test(line)) {
+                  tasks.push('☑')
+                }
+              })
+              return tasks
+            }
+
+            const parseLinks = text => {
+              const links = {}
+              text.split('\n').forEach(line => {
+                const linkMap = {
+                  draft: /^\[~draft-doc\]:\s*(?<link>\S+)/,
+                  sql: /^\[~sql-dir\]:\s*(?<link>\S+)/,
+                  results: /^\[~results-sheet\]:\s*(?<link>\S+)/
+                }
+                for (const [k, v] of Object.entries(linkMap)) {
+                  const m = line.match(v)
+                  if (m) {
+                    links[k] = m.groups.link
+                  }
+                }
+              })
+              return links
+            }
+
+            const parseTeam = text => {
+              const team = {}
+              text.split('\n').forEach(line => {
+                if (/\|\s*\[Sheet\]\[~results-sheet\]\s*\|/.test(line)) {
+                  const [, authors, reviewers, analysts] = line.split('|')
+                  team.authors = [...authors.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.reviewers = [...reviewers.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.analysts = [...analysts.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                }
+              })
+              return team
+            }
+
+            const formatLinks = links => {
+              const icons = {
+                draft: '🗎',
+                sql: '⛁',
+                results: '⚝'
+              }
```

Comment:
Let's use some more meaningful icons. As using the same and only showing when they exist makes it difficult to tell which ones you are showing. Plus the one you've chosen is very difficult to read on windows and just a blank square on macOS.

```suggestion
              const icons = {
                draft: '✍',
                sql: '🔍',
                results: '📊'
              }
```

Reply:
I chose all three icons different, but if it was not rendering on your machine then it perhaps is a font issue. Anyways, I am updating them.

Reply:
Ah. That explains why I saw one of the icons in Windows but none in Mac or iOS! Didn’t check the source to see if they all rendered and were different.

Above ones seem to be supported in both Apple and Windows world.

Reply:
I'm still seeing that first one as the default square on my machine:

![Screenshot of commit](https://user-images.githubusercontent.com/10931297/87204064-d5563d00-c2fb-11ea-8e09-0476d3eec6f9.png)

It's better now as at least they are different but the square is ugly. Can you see the one I suggested on your machine? If so, could we use that?

Reply:
> Above ones seem to be supported in both Apple and Windows world.

I am using Linux and they all rendered well here, including those that you suggested.

Reply:
OK but looks like you didn't use my suggested one for the first one as not rendering for me?

Reply:
> Can you see the one I suggested on your machine?

Yes!

> If so, could we use that?

Done!

Reply:
All good now - thanks!

Reply:
> OK but looks like you didn't use my suggested one for the first one as not rendering for me?

That's right, I tried something else which was more meaningful (a document with image and text), but when you reported it was not rendering well for you then I updated it to what you suggested.